ddtrace/tracer/option.go

```diff
@@ -429,7 +429,7 @@ func newConfig(opts ...StartOption) *config {
 	c.runtimeMetricsV2 = internal.BoolEnv("DD_RUNTIME_METRICS_V2_ENABLED", false)
 	c.debug = internal.BoolVal(getDDorOtelConfig("debugMode"), false)
 	c.logDirectory = os.Getenv("DD_TRACE_LOG_DIRECTORY")
-	c.enabled = newDynamicConfig("tracing_enabled", internal.BoolVal(getDDorOtelConfig("enabled"), true), func(b bool) bool { return true }, equal[bool])
+	c.enabled = newDynamicConfig("tracing_enabled", internal.BoolVal(getDDorOtelConfig("enabled"), true), func(_ bool) bool { return true }, equal[bool])
```

Comment:
@mtoffl01 You were the last to edit this line lol; do you know why we always return `true` for the third param here? Do we want to use the input parameter `b`? Same goes with line 1202 (`WithTraceEnabled()`)

Reply:
For reference, the param is `apply` which "executes any config-specific operations to propagate the update properly, returns whether the update was applied"